### ISSUE TITLE
:wheelchair: a11y(bal-dropdown): dropdown options are not accessible

### ISSUE THREAD

**Issue by zsofia-oroszi-ibmix** [baloise/design-system#1531]
### Description of this issue

**Keyboard access:**

- Selection with space not possible. Only enter.

**Screenreader access (VoiceOver + Chrome):**

- Input is read as textfield and therefore treated as one
- Since it is recognised as textfield opening dropdown with VO+space is not possible.

### How to fix it

-

### Code Reproduction URL

https://dev.author.baloise.magnolia-platform.io/webstyleguide-baloise-com/de/home/components/Formularbaukasten/Dropdown.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hirsch88** [comment]
Hi @zsofia-oroszi-ibmix 

Space will not be provided
Pressing Enter will select a highlighted option and collapse the list. (If nothing is highlighted in the list, pressing Enter will toggle the combo box open and closed.) Space cannot be used for selecting, as pressing it will submit a space character into the filter string.

TextField
Since autofill is a requirement we cannot use a button
